service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/NoticeOfProceedingsEmailContentProvider.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.notify.allocatedjudge.AllocatedJudgeTemplateForNoticeOfProceedings;
+import uk.gov.hmcts.reform.fpl.service.HearingBookingService;
+import uk.gov.hmcts.reform.fpl.service.email.content.base.AbstractEmailContentProvider;
+
+import java.time.format.FormatStyle;
+
+import static java.util.Objects.isNull;
+import static org.apache.commons.lang.StringUtils.capitalize;
+import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.formatLocalDateToString;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class NoticeOfProceedingsEmailContentProvider extends AbstractEmailContentProvider {
+    private final ObjectMapper mapper;
+    private final HearingBookingService hearingBookingService;
+
+    public AllocatedJudgeTemplateForNoticeOfProceedings buildAllocatedJudgeNotification(CaseDetails caseDetails) {
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
+
+        AllocatedJudgeTemplateForNoticeOfProceedings allocatedJudgeTemplate
+            = new AllocatedJudgeTemplateForNoticeOfProceedings();
+
+        allocatedJudgeTemplate.setFamilyManCaseNumber(isNull(caseData.getFamilyManCaseNumber()) ? ""
+                : caseData.getFamilyManCaseNumber() + ",");
+        allocatedJudgeTemplate.setLeadRespondentsName(capitalize(caseData.getRespondents1()
+                .get(0)
+                .getValue()
+                .getParty()
+                .getLastName()));
```

Comment:
We could probably use `PeopleInCaseHelper.getFirstRespondentLastName` here no? Also param should be setLeadRespondentLastName

Reply:
yes changed